steps/test/unit/tReorderMSData.cc

```diff
@@ -22,56 +22,51 @@ using dp3::steps::Step;
 
 BOOST_AUTO_TEST_SUITE(reorder)
 
-bool compareMetaBinaries(const std::string& filename1,
-                         const std::string& filename2) {
-  std::ifstream file1(
-      filename1,
-      std::ifstream::ate | std::ifstream::binary);  // open file at the end
-  std::ifstream file2(
-      filename2,
-      std::ifstream::ate | std::ifstream::binary);  // open file at the end
-
-  if (file1.fail() || file2.fail()) {
-    std::cout << "Failed to open Meta file" << std::endl;
-  }
+// Function to compare two binary files. If the files for comparison are Meta file, the flag isMetaFile is set to true.
+bool compareBinaryFiles(const std::string& referenceFile, const std::string& inputFile, bool isMetaFile) {
```

Comment:
For new code (like everything in this file) please adhere to naming conventions from google c++ style guide https://google.github.io/styleguide/cppguide.html#Variable_Names

e.g. `input_file` not `inputFile` please pay attention also to the rule of `Minimize the use of abbreviations`  while reading over the style guide.

If modifying existing functions that use old conventions then you can in those cases (at your discretion, if it makes sense) stick to the existing style within those functions for consistency instead.